.github/workflows/ci.yml

```diff
@@ -11,12 +11,16 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
-
     strategy:
       matrix:
-        node-version: [14, 16]
-
+        node-version: [14, 16, 17]
+        system:
+          - os: ubuntu-20.04
+        include:
+          - node_version: 18
+            system:
+              os: ubuntu-22.04
+    runs-on: ${{ matrix.system.os }}
```

Comment:
Why Node v15 is not included in the `node-version` matrix?
And why is Node v18 is listed separately?

Reply:
I honestly don't remember why it wasn't included, but unless we see specific requests for v15 builds I'm going to just be adding newer versions to the support matrix moving forward. 

The v18 build needs a more recent version of ubuntu; I added it as an `include` so I could have it run on ubuntu-22.04 without changing the `os` of the rest of the build matrix.